src/utils/bind.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Faster version of Function.prototype.bind
+ */
+module.exports = function bind (fn, ctx/* , arg1, arg2 */) {
```

Comment:
This function got more complicated because it doesn't accommodate for passing more arguments to `bind`, i.e. `bind(fn, context, VALUE1, VALUE2)`.  So I had to update it. :/